lib/view_resource/filter.dart

```diff
@@ -122,6 +125,53 @@ class ResourceFilter {
   }
 }
 
+// Resonsible for uploading searches and filter options
+class HomeAnalytics {
```

Comment:
Should we group all of the analytics classes into one module? I think HomeAnalytics and UserAnalytics could be co-located.